asgi_correlation_id/middleware.py

```diff
@@ -67,6 +70,9 @@ def __post_init__(self) -> None:
         If Sentry is installed, propagate correlation IDs to Sentry events.
         If Celery is installed, propagate correlation IDs to spawned worker processes.
         """
+        if self.validators is None:
+            self.validators = [is_valid_uuid] if self.validate_header_as_uuid else []
```

Comment:
the `if self.validate_header_as_uuid` part keeps us backwards compatible, then we can drop this in v2.

Reply:
Could just add this as a default argument to the `validators` argument above?